app/Http/Controllers/StatusController.php

```diff
@@ -126,6 +126,8 @@ public static function getDashboard(User $user): Paginator {
                      ->select('statuses.*')
                      ->orderBy('train_checkins.departure', 'desc')
                      ->whereIn('user_id', $followingIDs)
+                     ->whereIn('visibility', [0, 2])
+                     ->orWhere('user_id', auth()->user()->id)
```

Comment:
This function can (...) be used to get a dashboard for every user.
```suggestion
                     ->orWhere('user_id', $user->id)
```